### PR TITLE
Excessive logging

### LINKED ISSUE #388: Excessive Logging
## Is your feature request related to a problem? Please describe

Patron logs excessively while it is starting up and during operations.
A lot of these logs can be switch from `info` to `debug` in order to show them only id `debug` is requested and not all the time.

## Is this a bug? Please provide steps to reproduce, a failing test etc.

Not a bug.

## Describe the solution

Switched some logs from `info` to `debug`.

### PR DESCRIPTION
**Pull request by mantzas**
## Which problem is this PR solving?

Closes #388.

## Short description of the changes

Switched from `info` to `debug`.

### CODE REVIEW

**Review by drakos74 — DISMISSED** [review]

**Review by tpaschalis — DISMISSED** [review]

File: component/async/kafka/group/group.go [hunk 1]
```diff
@@ -114,7 +114,7 @@ func (c *consumer) Consume(ctx context.Context) (<-chan async.Message, <-chan er
 		return nil, nil, fmt.Errorf("failed to create consumer: %w", err)
 	}
 	c.cg = cg
-	log.Infof("consuming messages from topics '%s' using group '%s'", strings.Join(c.topics, ","), c.group)
+	log.Debugf("consuming messages from topics '%s' using group '%s'", strings.Join(c.topics, ","), c.group)
```
Comment (drakos74) [inline component/async/kafka/group/group.go:117]: consider keeping this, as it confirms the working of the component. It s very useful in debugging issues in prod.

File: component/async/kafka/simple/simple.go [hunk 2]
```diff
@@ -124,7 +122,7 @@ func (c *consumer) Consume(ctx context.Context) (<-chan async.Message, <-chan er
 	chMsg := make(chan async.Message, c.config.Buffer)
 	chErr := make(chan error, c.config.Buffer)
 
-	log.Infof("consuming messages from topic '%s' without using consumer group", c.topic)
+	log.Debugf("consuming messages from topic '%s' without using consumer group", c.topic)
```
Comment (drakos74) [inline component/async/kafka/simple/simple.go:125]: same as above

File: component/http/component.go [hunk 3]
```diff
@@ -68,11 +68,11 @@ func (c *Component) Run(ctx context.Context) error {
 
 func (c *Component) listenAndServe(srv *http.Server, ch chan<- error) {
 	if c.certFile != "" && c.keyFile != "" {
-		log.Infof("HTTPS component listening on port %d", c.httpPort)
+		log.Debugf("HTTPS component listening on port %d", c.httpPort)
 		ch <- srv.ListenAndServeTLS(c.certFile, c.keyFile)
 	}
 
-	log.Infof("HTTP component listening on port %d", c.httpPort)
+	log.Debugf("HTTP component listening on port %d", c.httpPort)
```
Comment (drakos74) [inline component/http/component.go:75]: sae as above, consider keeping this as info level

File: component/kafka/group/component.go [hunk 4]
```diff
@@ -193,7 +193,7 @@ func (c *Component) processing(ctx context.Context) error {
 		}
 
 		if client != nil {
-			log.Infof("consuming messages from topics '%#v' using group '%s'", c.topics, c.group)
+			log.Debugf("consuming messages from topics '%#v' using group '%s'", c.topics, c.group)
```
Comment (drakos74) [inline component/kafka/group/component.go:196]: same as above , consider keeping this to assess the components health

File: service.go [hunk 5]
```diff
@@ -90,7 +90,7 @@ func (s *service) createHTTPComponent() (Component, error) {
 		}
 	}
 	port = strconv.FormatInt(portVal, 10)
-	log.Infof("creating default HTTP component at port %s", port)
+	log.Debugf("creating default HTTP component at port %s", port)
```
Comment (drakos74) [inline service.go:94]: same as above

**Review by drakos74 — DISMISSED** [review]

File: component/async/kafka/group/group.go (hunk 1 above)
Comment (mantzas) [reply component/async/kafka/group/group.go:117]: We have prometheus metrics for this, would that be enough? checking the /metrics endpoint or the grafana dashboard should reveal everything.

**Review by tpaschalis — APPROVED** [review]

File: service.go (hunk 5 above)
Comment (mantzas) [reply service.go:94]: There is already a requested change at the HTTP component level that logs this. If we move forward with the changes we should have only one of those logs.

Comment (drakos74) [reply service.go:94]: ok, that makes sense. Missed the other one

File: component/async/kafka/group/group.go (hunk 1 above)
Comment (drakos74) [reply component/async/kafka/group/group.go:117]: ok, sounds good as well.

**Review by drakos74 — APPROVED** [review]

**Pull request merged**